boards/xtensa/heltec_wifi_lora32/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_BOOTLOADER_ESP_IDF)
```

Comment:
Is it necessary to duplicate this file? Maybe we can create a common CMake file which can be used by all ESP32 boards.

Reply:
@shubhamkulkarni97 you're correct. I copied it from the esp32 directory but I verified there's no problem removing it